### ISSUE TITLE
(Enhancement Request) MeshRouter grouping of connections

### ISSUE THREAD

**Issue by D4V3M0NK** [Ylianst/MeshCentral#352]
I think MCR would benefit from mimicking the groups that are present on MC, so that upon opening MCR, you're presented with the groups that the maps are associated to, rather than just a (potentially) long list of connections. When you select the group that you're interested in, then you should see the maps associated to that group and be able to Open / Remove them as necessary.

**Comment by Ylianst** [comment]
Agreed. I am a bit over worked these days, there are more feature requests than I can handle, but this is a good request.
